.circleci/config.yml

```diff
@@ -254,7 +254,7 @@ jobs:
           name: Install requirements
           command: |
             pip install -r requirements.txt
-            pip install -e ./pyodide-build
+            pip install pyodide-build --upgrade
```

Comment:
What happens if we want to test against a fork of `pyodide-build`? Can we add an empty constraints file called `pyodide-build-version.txt` and pass `-c pyodide-build-version.txt` in CI?

Reply:
Perhaps even add `pyodide-build` to `requirements.txt` and do:
```
pip install -r requirements.txt -c pyodide-build-version.txt
```

Reply:
That sounds reasonable to me.

Reply:
Currently, the requirements.txt file is used to specify test dependencies, so it doesn't seem like a good idea to put build dependencies in there. Let me think of a better way.